### DISCUSSION TITLE
Shared online lists

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-AutoUpdate#147] [Ideas]
Blacklist/Whitelist via URL (like we handle our IE-mode list for Edge in GPO making both on-/offsite access possible for clients).

- Install with parameter -SharedList https://www.example.site/lists (save in registry)
- In Get-ExcludedApps/Get-IncludedApps test for the existent of files in URL
- If exist and modified date/time differs from local file, download to be used!
- Else use local file

**Comment by KnifMelti** [comment]
...maybe ADMX files for WAU for GPO (like winget itself) 🙏

**Comment by Romanitho** [comment]
This can be a nice feature :)
to be honest, I use Intune so quite easy to push updated lists ^^
Don't you use a cloud gateway with your SCCM ?

**Reply by KnifMelti** [reply]
Not in the moment, it is coming (but I'm not the responsible)...

**Reply by Romanitho** [reply]
haha. at least, it seems to be in the pipe :)

**Reply by KnifMelti** [reply]
Have a nice vacation!

It's definitively in the pipe...
...defined in SCCM, but not yet active.

BTW.. ..when you say "_../push updated lists/.._", I reckon in Intune (just decided to dig in to the **MEM admin** there, leaving **SCCM**) there's something like push a file like in GPO?
Using what as a detection for that the new file (is it versioned in the Intune push itself?) has been saved locally?

Is there a way to use an UNC path or a HTTPS to/via the cloud gateway?

I think that a shared online list with WAU checking for it automatically (like Edge/IE-mode) is a better way.
Then you can just forget about the pushing of a list and just edit the shared list itself.

I today rolled out **Company Portal** via SCCM to a selected few hybrid joined clients because of 1.3 stable is now out, the latest news from MS (that winget application is coming in Intune/MEM admin and you can mark the application for automatic upgrade) and the need for Company Portal on clients for managing via MEM.

So, even a dinosaur can learn ;)

**Reply by KnifMelti** [reply]
https://techcommunity.microsoft.com/t5/windows-it-pro-blog/update-to-endpoint-manager-integration-with-the-microsoft-store/ba-p/3585077

**Reply by KnifMelti** [reply]
...and maybe a parameter when installing to add repositories hosted internally (privately) for an organization?

**Reply by Romanitho** [reply]
For one of my customer for example, we have this scenario:
- WAU without self-update
- White List activated
- Update at logon and once a week by default

The trick we use with Intune to update the white list is to add a subversion number to WAU version. For example, the current WAU stable version is 1.13.2 so my WAU "costumed" version will be 1.13.2.1 and I edit the version inside the WAU install script. This way, each time my customer edit his whitelist, he add the App IDs in the list and package a new WAU version in Intune that will be 1.13.2.2 etc etc... and adjust the detection method to match the new version. As WAU is very light, it will be reinstalled with the new whitelist. In their context, they update the white list very occasionally. Once another WAU version is release, they package it with their latest whitelist and change the number to 1.14.0.1 and so on.

Another approach could be to package the list only and use detection method on last edit file date 😅

**Reply by KnifMelti** [reply]
#166
